artview/components/select_region.py

```diff
@@ -218,48 +230,40 @@ def CreateSelectRegionWidget(self):
         self.setCentralWidget(self.SelectRegionbox)
 
         # Add buttons for functionality
-        self.buttonViewTable = QtGui.QPushButton('View Tabular Data', self)
-        self.buttonViewTable.setToolTip("View Region Data in popup window")
-        self.buttonOpenTable = QtGui.QPushButton('Open Tabular Data', self)
-        self.buttonOpenTable.setToolTip("Open a Region Data CSV file")
-        self.buttonSaveTable = QtGui.QPushButton('Save Tabular Data', self)
-        self.buttonSaveTable.setToolTip("Save a Region Data CSV file")
-        self.buttonStats = QtGui.QPushButton('Stats', self)
-        self.buttonStats.setToolTip("Show basic statistics of selected Region")
-        self.buttonHist = QtGui.QPushButton('Plot Histogram', self)
-        self.buttonHist.setToolTip("Plot histogram of selected Region")
-        self.buttonApplyMask = QtGui.QPushButton('Mask Region', self)
-        self.buttonApplyMask.setToolTip("Apply filter mask to selected Region")
-        self.buttonRestoreMask = QtGui.QPushButton('Restore Mask', self)
-        self.buttonRestoreMask.setToolTip("Restore the original data mask")
         self.buttonResetSelectRegion = QtGui.QPushButton('Reset Region', self)
         self.buttonResetSelectRegion.setToolTip("Clear the Region")
-        self.saveButton = QtGui.QPushButton("Save File", self)
-        self.saveButton.setToolTip("Save modified radar instance to cfradial file") 
         self.buttonHelp = QtGui.QPushButton('Help', self)
         self.buttonHelp.setToolTip("About using SelectRegion")
-        self.buttonViewTable.clicked.connect(self.viewTable)
-        self.buttonOpenTable.clicked.connect(self.openTable)
-        self.buttonSaveTable.clicked.connect(self.saveTable)
-        self.buttonStats.clicked.connect(self.displayStats)
-        self.buttonHist.clicked.connect(self.showHist)
-        self.buttonApplyMask.clicked.connect(self.applyMask)
-        self.buttonRestoreMask.clicked.connect(self.restoreMask)
+        self.buttonBack = QtGui.QPushButton('Back', self)
+        self.buttonBack.setToolTip("Remove last Polygon")
```

Comment:
It depends how much time one spend creating a polygon, if it is to quick that is not needed, otherwise we could implement. As always I'm open for name suggestions.